server/app/controllers/api_controller.rb

```diff
@@ -1,7 +1,7 @@
 class ApiController < ApplicationController
   rescue_from ActionController::ParameterMissing, with: :render_exception
 
-  def render_exception(exception)
-    render :json => {}, :status => :unprocessable_entity
+  def render_exception(_exception)
```

Comment:
Why are we even taking an exception here if we aren't doing anything with it?

Reply:
If I don't add that here then I get an `ArgumentError` since the exception is passed in as an argument.